test/bin/build_images.sh

```diff
@@ -381,7 +382,31 @@ do_group() {
         done
     fi
 
-    if (( ${#buildid_list[@]} )); then
+    if ${BUILD_INSTALLER} && ! ${COMPOSER_DRY_RUN}; then
+        for download_file in "${groupdir}"/*.image-fetcher; do
+            local download_url
+            download_url=$("${GOMPLATE}" --file "${download_file}")
+            blueprint="$(basename -s .image-fetcher "${download_file}")"
+            local expected_iso_file="${VM_DISK_BASEDIR}/${blueprint}.iso"
+            if [ -f "${expected_iso_file}" ]; then
+                echo "${expected_iso_file} already exists"
+                if should_skip "${blueprint}"; then
+                    record_junit "${groupdir}" "${download_file}" "download" "SKIPPED"
+                    continue
+                fi
+            fi
+
+            echo "Adding image-fetcher for ${download_file}"
+            download_opts+=("${expected_iso_file} ${download_url}")
+        done
+    fi
+
+    # Run image-fetcher while osbuilder is running in background
+    if [ ${#download_opts[@]} -ne 0 ]; then
+        echo "Waiting for image-fetcher to complete..."
+        parallel --colsep ' ' wget -nv -O "{1}" "{2}" ::: "${download_opts[@]}"
```

Comment:
The build_rpms.sh script uses some additional options to provide progress information when running on the console and to save log output to files for debugging.